src/lib/layout/layout.component.ts

```diff
@@ -98,9 +113,26 @@ export class PlexLayoutComponent implements AfterContentInit, OnInit, OnDestroy
         }
     }
 
-
-    constructor() {
+    // Sidebar expandible
+    public contraer(steps) {
+        steps = this.steps;
```

Comment:
Creo que esto quedó de la otra revisión, es al revés la asignación o no va directamente?
```suggestion
        this.steps = steps;
```

Reply:
No iba directamente. Ahi lo corregí. Gracias @andrrrl